Install via conda environment file

niksirbi opened the pull request:
Hey @viktorpm, because we have many conda dependencies, I've included a conda environment file and updated the installation instructions accordingly. It would be great if you could test whether this works on your side.

viktorpm commented on README.md:
```diff
@@ -26,25 +26,32 @@ Build unbiased anatomical templates from individual images
-conda create -n template-builder -c aramislab -c conda-forge python=3.10 ants parallel
+git clone https://github.com/brainglobe/brainglobe-template-builder
+cd brainglobe-template-builder
+conda env create -f environment.yaml -n template-builder
```
I started from scratch: removed the existing environment, cloned the repo and manually downloaded the environment.yaml file. The environment was not created. I got this message: 
`Collecting package metadata (repodata.json): done
Solving environment: \ Killed`

viktorpm reviewed (commented):
Couldn't create conda environment

adamltyson commented:
FYI this works for me. @viktorpm are you using Windows?

viktorpm commented:
> FYI this works for me. @viktorpm are you using Windows?

@adamltyson, I'm using WSL but I could install everything previously

niksirbi commented:
> I'm using WSL but I could install everything previously

What has also changed is that I have added `napari` as a dependency. Maybe that is causing a dependency resolution conflict. Did `conda` take very long to solve the environment?

niksirbi commented:
@viktorpm Unfortunately this tends to happen sometimes, `conda` is sometimes very slow to resolve dependencies. Maybe it's a good opportunity to install [mamba](https://mamba.readthedocs.io/en/latest/mamba-installation.html#mamba-install), which is basically the same as conda, but much faster (you can use all the same commands as before, just replace `conda` with `mamba`). 

If this solves the problem, I might update the instructions to recommend `mamba` as the default option.
